packages/@webex/plugin-meetings/src/meeting/locusMediaRequest.ts

```diff
@@ -250,12 +251,19 @@ export class LocusMediaRequest extends WebexPlugin {
       this.confluenceState = 'creation in progress';
     }
 
-    // @ts-ignore
-    return this.request({
+    const upload = new EventEmitter();
+    const download = new EventEmitter();
+
+    const options = {
       method: HTTP_VERBS.PUT,
       uri,
       body,
-    })
+      upload,
```

Comment:
maybe we should do this extra logging only when `(request.type === 'RoapMessage')`, because LocusMediaRequest is also used for mute operations and these won't be interesting to us and can fill up the log